Composer/packages/client/src/pages/publish/BotStatusList.tsx

```diff
@@ -48,6 +49,25 @@ export const BotStatusList: React.FC<BotStatusListProps> = ({
 }) => {
   const [expandedBotIds, setExpandedBotIds] = useState<Record<string, boolean>>({});
   const [currentSort, setSort] = useState({ key: 'Bot', descending: true });
+  const [clipboardText, setClipboardText] = useState('');
+  const clipboardTextFieldRef = useRef<ITextField>(null);
+
+  const copyStringToClipboard = (value?: string) => {
+    try {
+      if (clipboardTextFieldRef.current) {
```

Comment:
I dont think you should do the copy command if the text is empty.

Reply:
I'll be moving all this code to a smarter component Monday.